Formula/azure-cli.rb

```diff
@@ -3,17 +3,12 @@ class AzureCli < Formula
 
   desc "Microsoft Azure CLI 2.0"
   homepage "https://docs.microsoft.com/cli/azure/overview"
-  url "https://codeload.github.com/Azure/azure-cli/legacy.tar.gz/814598f5e8fe3d0e418f0be392a0f2984b6bd215"
-  version "2.11.1"
-  sha256 "6de9b5c0f7b03b2250ea4b6424dfafd5c7d134e7b82d47a52815ea46dabd9a68"
+  url "https://codeload.github.com/Azure/azure-cli/legacy.tar.gz/39d334dddd25224b97eacac39c12f39d41b902c1"
+  version "2.12.0"
+  sha256 "db7bf216db0adc25aa9e895d01e1aa23f4983e4f04633d757482ef9f67cbc987"
   license "MIT"
   head "https://github.com/Azure/azure-cli.git"
 
-  livecheck do
-    url "https://github.com/Azure/azure-cli/releases/latest"
-    regex(%r{href=.*?/tag/azure-cli[._-]v?(\d+(?:\.\d+)+)["' >]}i)
-  end
-
```

Comment:
The `livecheck` block should not be removed.

Reply:
Added again. Thanks.